chef/cookbooks/provisioner/templates/default/crowbar_join.suse.sh.erb

```diff
@@ -506,8 +490,8 @@ if [ "$MODE" == "setup" -o "$MODE" == "start" ]; then
 
     [ $final_state == "ready" ] || EXVAL=1
 elif [ "$MODE" == "stop" ]; then
-    HOSTNAME=$(hostname -f)
     state=$(get_state $HOSTNAME)
+    HOSTNAME=$(hostname -f)
```

Comment:
Why this change?

Reply:
Because `crowbarcrl node` (that gets executed in `get_state`) does not expect node's fqdn AFAIK

Reply:
Ah, it may make sense to add a comment about it for the future.